thirdparty/build-thirdparty.sh

```diff
@@ -767,6 +735,15 @@ build_hyperscan() {
     make -j$PARALLEL && make install
 }
 
+#mariadb-connector-c
+build_mariadb() {
+    check_if_source_exist $MARIADB_SOURCE
+    cd $TP_SOURCE_DIR/$MARIADB_SOURCE
+    mkdir -p build && cd build
+    cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=${TP_INSTALL_DIR}
```

Comment:
Better to set '-DCMAKE_INSTALL_LIBDIR=lib64' ?

Reply:
mariadb-connector-c cmake don't have variable CMAKE_INSTALL_LIBDIR